docs/device_support_guide.md

```diff
@@ -0,0 +1,99 @@
+# Intro
+There are two main methods to get your MSI laptop supported: the recommended method requires Windows to be installed, and the other works directly on Linux.
+
+If there are any BIOS/firmware updates available for your laptop, follow this guide for your current firmware before installing any of them. Then repeat the process for each new firmware version you install from the official MSI website. This is required to obtain support for older firmware as the EC configuration may vary across the versions.
+
+## Windows method (recommended):
+
+1. Install Windows 10/11 normally, booting directly from a live usb or
+any other trick won't work, however windows activation is not
+needed.
+
+2. After a successful installation, make sure to download/install all
+Windows updates from the settings
+(windows can't update bios/firmware automatically on MSI
+laptops but that doesn't mean you have to update them yourself,
+instead continue following the guide).
+
+3. Install the MSI app designed for your laptop: MSI dragon center /
+MSI creator center / MSI center / MSI center pro (the correct app
+can be found on the support page for your laptop).
+
+4. If you fail at step 3, then you most likely need another app. usually
+its AMD Adrenalin software / GeForce Experience (nvidia)/ or intel
+equivalent.
+
+5. Once the MSI app is installed, you can test the functions it offers,
+like user scenario and battery charge limit.
+
+6. If everything works as expected, download RWEverything:
+https://rweverything.com/download/ ![download RWEverything](pics/support_guide/dl_rwe.png)
+
+7.  Launch it as administrator:
+![run as admin](pics/support_guide/run_as_admin.png)
+
+8. Navigate to the EC tab (page):
+![open ec tab](pics/support_guide/open_ec_tab.png)
+
+9. Here you should see a table of all the values your Embedded Chip has in its memory.
+
+**Do not attempt to edit the memory manually. Writing the wrong the values to the wrong addresses will DAMAGE YOUR HARDWARE or even BRICK YOUR DEVICE! This can't be reversed by resetting the EC or BIOS.**
```

Comment:
```suggestion
> [!CAUTION]
**Do not attempt to edit the memory manually. Writing the wrong the values to the wrong addresses will DAMAGE YOUR HARDWARE or even BRICK YOUR DEVICE! This can't be reversed by resetting the EC or BIOS.**
```